### DISCUSSION TITLE
Play in a react app

### DISCUSSION THREAD

**Discussion by ghost** [streamlink/streamlink#3581] [Q&A]
I plan to create a simple reactJS app to play videos from different sites.
Can I use streamlink to feed a player on a react app?

Also, which react player you recommend?
I'm a noob here

**Comment by gravyboat** [comment]
> Can I use streamlink to feed a player on a react app?

You might be able to using something like https://streamlink.github.io/cli.html#cmdoption-player-external-http or look at how https://github.com/streamlink/streamlink-twitch-gui handles it but we're not going to provide support, suggestions, or advice on how to do this. It's way outside the scope of the project. You'll have to figure it out on your own.
